tool/jt.rb

```diff
@@ -750,6 +752,42 @@ def cextc(cext_dir, test_gem=false, *clang_opts)
     end
   end
 
+  def pr(*args)
+    out, _err   = raw_sh('git remote', capture: true)
+    remotes     = out.split
+    remote_urls = remotes.map do |r|
+      out, _err = raw_sh("git config --get remote.#{r}.url", capture: true)
+      [r, out.chomp!]
+    end
+
+    upstream = remote_urls.find { |r, u| u.include? 'graalvm/truffleruby' }.first
+    bb       = remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
+
+    pr_number = args.first
+    if pr_number
+      github_pr_branch = "#{upstream}/pr/#{pr_number}"
+    else
+      fetch     = "+refs/pull/*/head:refs/remotes/#{upstream}/pr/*"
+      out, _err = raw_sh("git config --get-all remote.#{upstream}.fetch", capture: true)
+      raw_sh "git config --add remote.#{upstream}.fetch \"#{fetch}\"" unless out.include? fetch
+      raw_sh "git fetch #{upstream}"
```

Comment:
What is this complex logic?
Shouldn't it be as simple as
```
git push ol my-branch
# Or
git push ol my-branch:github/pr/mybranch
```
I think the namespacing is not necessary and make branches longer than needed if this our primary way to develop.